src/Orleans/Messaging/Message.cs

```diff
@@ -263,6 +263,15 @@ public bool IsNewPlacement
             }
         }
 
+        public bool IsUsingIfaceVersion
+        {
+            get { return Headers.IsUsingIfaceVersion; }
```

Comment:
The abbreviation comment actually applied to all the IsUsingIfaceVersion properties also, and IS_USING_INTERFACE_VERSION.